### ISSUE TITLE
Uncertainty estimate for parameter fits

### ISSUE THREAD

**Issue by ductng** [scikit-hep/pyhf#642]
Hi,

I could not find any examples how to obtain the uncertainty of parameters in parameter fits. This might not be possible in all cases for all optimisers and methods, but it would be nice to have a few options for this.

E.g. did you consider interfacing to lmfit (https://lmfit.github.io/lmfit-py/) to automate the minimisation and error estimate for parameter fits?

Cheers
Duc

**Comment by lukasheinrich** [comment]
hi @ductng, when using the minuit optimizer it is possible to get the uncertainties of the best fit parameters

```
import pyhf
pyhf.set_backend(pyhf.tensor.numpy_backend(),pyhf.optimize.minuit_optimizer())
m = pyhf.simplemodels.hepdata_like([5],[50],[1]) 
d = [50] + m.config.auxdata
results = pyhf.optimizer._make_minuit(pyhf.utils.loglambdav,d,m,m.config.suggested_init(),m.config.suggested_bounds()).migrad()
print(result)
```

I'm working on a PR to make this have a nicer API. yes, we're looking at lmfit-py!

**Comment by lukasheinrich** [comment]
fixed here: #690
